jujugui/static/gui/src/app/components/model-actions/_model-actions.scss

```diff
@@ -14,12 +14,15 @@
     border: 1px solid $light-mid-grey;
     transition: background-color .2s;
     background-color: $canvas-background;
+    // scss-lint:disable ImportantRule
+    color: #888 !important;
```

Comment:
I think there is a variable for this colour. Is it vanillas fault that an `!important` is required here?